api/app.py

```diff
@@ -78,7 +78,11 @@ class DifyApp(Flask):
 def create_app() -> Flask:
     app = DifyApp(__name__)
     app.config.from_object(Config())
-
+    os.environ['REDIS_CLUSTER_ENABLED'] = app.config['REDIS_CLUSTER_ENABLED']
+    os.environ['REDIS_USERNAME'] = app.config['REDIS_USERNAME']
+    os.environ['REDIS_PASSWORD'] = app.config['REDIS_PASSWORD']
+    os.environ['REDIS_HOST'] = app.config['REDIS_HOST']
+    os.environ['REDIS_PORT'] = app.config['REDIS_PORT']
```

Comment:
It sounds kind of looped dependencies, as it's setting the configs from app configs to system envs, while the app configs are also read from system envs. No a good idea change system envs at runtime for satisfying the extention`ext_redis`'s need.

Reply:
@maldinixiang Please continue to update.